packages/expo-updates/package.json

```diff
@@ -39,7 +39,7 @@
     "preset": "expo-module-scripts"
   },
   "dependencies": {
-    "@expo/code-signing-certificates": "0.0.2",
```

Comment:
What's the reason for pinning exact dependency version?

Reply:
Didn't seem to upgrade until I pinned it exactly. Since it was already pinned before as well I figured I'd keep it pinned. That being said, I know very little about node dependency resolution, and even less about pre-1.0 semvers.